src/Dfe.PlanTech.Infrastructure.SignIn/Dfe.PlanTech.Infrastructure.SignIn.csproj

```diff
@@ -7,11 +7,11 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="Microsoft.AspNetCore.Authentication.OpenIdConnect" Version="7.0.5" />
-    <PackageReference Include="Microsoft.AspNetCore.Identity.UI" Version="7.0.5" />
+    <PackageReference Include="Microsoft.AspNetCore.Authentication.OpenIdConnect" Version="7.0.11" />
+    <PackageReference Include="Microsoft.AspNetCore.Identity.UI" Version="7.0.11" />
     <PackageReference Include="Microsoft.Extensions.Configuration.Abstractions" Version="7.0.0" />
     <PackageReference Include="Microsoft.Extensions.DependencyInjection.Abstractions" Version="7.0.0" />
-    <PackageReference Include="Microsoft.Identity.Abstractions" Version="3.2.1" />
+    <PackageReference Include="Microsoft.Identity.Abstractions" Version="5.0.0" />

```

Comment:
Can you try removing this package (`Microsoft.Identity.Abstractions`)? Pretty sure it's not needed

Reply:
done.